### ISSUE TITLE
[foxfooding] pressing the "What is multi-hop" arrow works to hide/show only once (iOS)

### ISSUE THREAD

**Issue by pdehaan** [mozilla-mobile/mozilla-vpn-client#3699]
In the page &raquo; Select location &raquo; Multi-hop pressing the "What is multi-hop" arrow works to hide/show only once. Pressing the arrow to reopen the explanation menu doesn't open the explanation anymore, you have to leave the page and come back to reset the state for it to work.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/VPN-2371)

**Comment by data-sync-user** [comment]
➤ Peter deHaan commented:

Row 47 in https://docs.google.com/spreadsheets/d/1nnhaCKuosXdIRtimbS6SwMvt-SqMWwTsjLoFnBKugO4/edit#gid=0 ( https://docs.google.com/spreadsheets/d/1nnhaCKuosXdIRtimbS6SwMvt-SqMWwTsjLoFnBKugO4/edit#gid=0|smart-link )

**Comment by data-sync-user** [comment]
➤ Maria Berlinger commented:

Thank you for logging this. The issue is already tracked here: https://mozilla-hub.atlassian.net/browse/VPN-2033 ( https://mozilla-hub.atlassian.net/browse/VPN-2033|smart-link )

**Comment by data-sync-user** [comment]
➤ Lesley Norton commented:

Confirming Maria Berlinger 's comment. Closing as dupe.

**Comment by data-sync-user** [comment]
➤ Lesley Norton commented:

Closing as duplicate of VPN-2033 which is confirmed by QA as fixed.
